src/com/example/powerup/Scenario.java

```diff
@@ -0,0 +1,55 @@
+package com.example.powerup;
+
+public class Scenario {
+
+	private Integer ID;
```

Comment:
Even if it's an accronim, use lowercase for all non-static variables (id) in a camelCase way and reserve uppercase for static variables